File: app/src/main/java/com/example/wuzzaf/activities/CandidatMain.java
package com.example.wuzzaf.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.wuzzaf.R;

public class CandidatMain extends AppCompatActivity implements View.OnClickListener {
    Button btnEmployCv,btnJobSuggest;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_candidat_main);
        btnEmployCv = (Button) findViewById(R.id.btnEmployCv);
        btnEmployCv.setOnClickListener(this);
        btnJobSuggest = (Button)findViewById(R.id.btnJobSuggest);
        btnJobSuggest.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        if (v != null) {
            switch (v.getId()) {
                case R.id.btnEmployCv:
                    Intent i = new Intent(this, EmployCvActivity.class);
                    startActivity(i);
                    finish();
                    break;

                case R.id.btnJobSuggest:
                    Intent intent = new Intent(this, JopSugisition.class);
                    startActivity(intent);
                    finish();
                    break;

            }
        }
    }
}

File: app/src/main/java/com/example/wuzzaf/activities/CandidatMain.kt
package com.example.wuzzaf.activities

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.wuzzaf.R
import com.example.wuzzaf.dao.EmployFirestoreManager
import com.example.wuzzaf.helpers.SharedPrefrenceHelper
import com.example.wuzzaf.helpers.ToastMessage

class CandidatMain : AppCompatActivity(), View.OnClickListener {
    private lateinit var employFirestoreManager: EmployFirestoreManager
    lateinit var userName: String
    lateinit var mContext: Context
    var btnEmployCv: Button? = null
    var btnJobSuggest: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_candidat_main)
        btnEmployCv = findViewById<View>(R.id.btnEmployCv) as Button
        btnEmployCv!!.setOnClickListener(this)
        btnJobSuggest = findViewById<View>(R.id.btnJobSuggest) as Button
        btnJobSuggest!!.setOnClickListener(this)
        employFirestoreManager = EmployFirestoreManager.newInstance()
        mContext = this

       // ToastMessage.addMessage(userName,this)
    }

    override fun onClick(v: View) {
        if (v != null) {
            when (v.id) {

                R.id.btnEmployCv -> {
                    val i = Intent(this, EmployCvActivity::class.java)
                    startActivity(i)
                    finish()
                }

                R.id.btnJobSuggest -> {
                    val intent = Intent(this, JopSugisition::class.java)
                    startActivity(intent)
                    finish()
                }
            }
        }
    }
}